src/dmd/cppmanglewin.d

```diff
@@ -500,30 +500,38 @@ private:
                 //d.toChars(), d.isVirtualMethod(), d.isVirtual(), cast(int)d.vtblIndex, d.interfaceVirtual);
             if (d.isVirtual() && (d.vtblIndex != -1 || d.interfaceVirtual || d.overrideInterface()))
             {
-                switch (d.protection.kind)
+                final switch (d.protection.kind)
                 {
                 case Prot.Kind.private_:
                     buf.writeByte('E');
                     break;
                 case Prot.Kind.protected_:
                     buf.writeByte('M');
                     break;
-                default:
+                case Prot.Kind.undefined:
+                case Prot.Kind.none:
+                case Prot.Kind.package_:
+                case Prot.Kind.public_:
+                case Prot.Kind.export_:
```

Comment:
Those look like uncovered cases. Anyone familiar with `cppmanglewin`?

Reply:
Nope, but this should have been a translation of the dmd backend mangler. I wouldn't expect C++ to mangle any of those anyway

Reply:
The coverage analyzer only runs on Linux, so Windows code won't get executed.

Reply:
Note that the mangler is performance critical. Adding all these extra cases that do nothing just slows it down.